python/mxnet/ndarray.py

```diff
@@ -1410,19 +1449,20 @@ def power(base, exp):
     # pylint: enable= no-member, protected-access
 
 def maximum(lhs, rhs):
-    """Element-wise maximum of array elements with broadcasting.
+    """Returns element-wise maximum of the array elements with broadcasting.
```

Comment:
no need to say array elements when you already returns element-wise.
Follow a consistent pattern across summary lines. Lets call input arrays/input instead of arguments or lhs/rhs.
---
Returns element-wise maximum of the input arrays with broadcasting.